### PR TITLE
Add summer school 2021-09

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: meetings/2021-09/program.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Program
+* Start: Monday, September 6
+* Finish: Friday, September 10
+
+The aim of the summer school is to teach participants knowledge
+about computational algebra, both theoretical and practical. To this end,
+we plan the following:
+- introductory courses by experts on select topics of computational algebra
+- introductory sessions on working with git, Julia, OSCAR
+- hand-on training sessions implementing select algorithms in Julia and Oscar
```
Comment (ThomasBreuer) [inline meetings/2021-09/program.md]: "selected algorithms"?

File: meetings/2021-09/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+layout: meeting
+title: Summer School 09/2021
+meeting: true
+meeting_nr: 202109
+is_meeting_index: true
+---
+
+# Summer School “Computeralgebra”
```
Comment (ThomasBreuer) [inline meetings/2021-09/index.md]: "Computer algebra"?

File: meetings/2021-09/participants.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+layout: meeting
+title: Participants
+meeting: true
+meeting_nr: 202109
+people:
+    - {name: Claus Fieker, affiliation: TU Kaiserslautern}
+    - {name: Max Horn, affiliation: TU Kaiserslautern}
+    - {name: Thomas Breuer, affiliation: RWTH Aachen}
+    - {name: Tommy Hofmann, affiliation: Uni Saarbrücken}
```
Comment (thofma) [inline meetings/2021-09/participants.md]:
```suggestion
    - {name: Tommy Hofmann, affiliation: Saarland University}
```

File: meetings/2021-09/program.md (hunk 1 above)
Comment (thofma) [inline meetings/2021-09/program.md]:
```suggestion
- hand-on training sessions implementing selected algorithms in Julia and Oscar
```

**Review by ThomasBreuer — APPROVED** [review]
just two comments

**Review by thofma — APPROVED** [review]

File: meetings/2021-09/index.md (hunk 2 above)
Comment (fingolfin) [inline meetings/2021-09/index.md]:
```suggestion
# Summer School “Computer algebra”
```

File: meetings/2021-09/program.md (hunk 1 above)
Comment (fingolfin) [inline meetings/2021-09/program.md]:
```suggestion
- hand-on training sessions implementing selected algorithms in Julia and Oscar
```

**Pull request merged**